tests/NLog.UnitTests/Config/ServiceRepositoryTests.cs

```diff
@@ -44,51 +46,81 @@ public class ServiceRepositoryTests : NLogTestBase
         [Fact]
         public void SideBySideLogFactoryExternalInterfaceTest()
         {
-            // Stage
-            LogFactory logFactory1 = new LogFactory();
-            logFactory1.ServiceRepository.RegisterType(typeof(IMyPrettyInterface), (t) => new MyPrettyImplementation() { Test = nameof(logFactory1) });
+            // Arrange
+            var logFactory1 = new LogFactory();
+            const string name1 = "name1";
+            logFactory1.ServiceRepository.RegisterType(typeof(IMyPrettyInterface), (t) => new MyPrettyImplementation() { Test = name1 });
```

Comment:
@snakefoot PS; the `nameof` confused me a lot here. So create a separate variable "name")